linera-rpc/src/client.rs

```diff
@@ -245,6 +245,28 @@ impl ValidatorNode for Client {
         })
     }
 
+    async fn download_certificates_by_heights(
+        &self,
+        chain_id: ChainId,
+        mut heights: Vec<BlockHeight>,
+    ) -> Result<Vec<ConfirmedBlockCertificate>, NodeError> {
+        heights.sort();
```

Comment:
We could sort already in `remote_node`. That would make it easier to immediately check that we got the certificates we requested.